Description: Fix bashisms in shell scripts.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-03-23

--- magnum-2.0.0.orig/magnum/templates/kubernetes/fragments/enable-kube-podmaster.sh
+++ magnum-2.0.0/magnum/templates/kubernetes/fragments/enable-kube-podmaster.sh
@@ -2,7 +2,7 @@
 
 . /etc/sysconfig/heat-params
 
-function init_templates {
+init_templates () {
     local TEMPLATE=/etc/kubernetes/manifests/kube-podmaster.yaml
     [ -f ${TEMPLATE} ] || {
         echo "TEMPLATE: $TEMPLATE"
@@ -58,7 +58,7 @@ EOF
     local SERVICE_ACCOUNT_PRIVATE_KEY_FILE=/etc/kubernetes/ssl/server.key
     local ROOT_CA_FILE=/etc/kubernetes/ssl/ca.crt
 
-    if [ "${TLS_DISABLED}" == "True" ]; then
+    if [ "${TLS_DISABLED}" = "True" ]; then
         SERVICE_ACCOUNT_PRIVATE_KEY_FILE=
         ROOT_CA_FILE=
     fi
--- magnum-2.0.0.orig/magnum/templates/kubernetes/fragments/enable-kube-proxy-master.sh
+++ magnum-2.0.0/magnum/templates/kubernetes/fragments/enable-kube-proxy-master.sh
@@ -2,7 +2,7 @@
 
 . /etc/sysconfig/heat-params
 
-function init_templates {
+init_templates () {
     local TEMPLATE=/etc/kubernetes/manifests/kube-proxy.yaml
     [ -f ${TEMPLATE} ] || {
         echo "TEMPLATE: $TEMPLATE"
--- magnum-2.0.0.orig/magnum/templates/kubernetes/fragments/enable-kube-proxy-minion.sh
+++ magnum-2.0.0/magnum/templates/kubernetes/fragments/enable-kube-proxy-minion.sh
@@ -2,10 +2,10 @@
 
 . /etc/sysconfig/heat-params
 
-function init_templates {
+init_templates () {
     local KUBE_PROTOCOL="https"
     local KUBE_CONFIG="/srv/kubernetes/kubeconfig.yaml"
-    if [ "${TLS_DISABLED}" == "True" ]; then
+    if [ "${TLS_DISABLED}" = "True" ]; then
         KUBE_PROTOCOL="http"
         KUBE_CONFIG=
     fi
